testing/mysql/integration_test.go

```diff
@@ -250,6 +252,20 @@ func checkBigInt(ctx context.Context, t *testing.T, client *spanner.Client) {
 	}
 }
 
+func checkJson(ctx context.Context, t *testing.T, dbURI string) {
+	resp, err := databaseAdmin.GetDatabaseDdl(ctx, &databasepb.GetDatabaseDdlRequest{Database: dbURI})
+	if err != nil {
+		t.Fatalf("Could not read DDL from database %s: %v", dbURI, err)
+	}
+	for _, stmt := range resp.Statements {
+		if strings.Contains(stmt, "CREATE TABLE customers") && strings.Contains(stmt, "customer_profile JSON") {
+			return
+		}
+	}
+	t.Fatal("cannot find converted json data type in spanner schema")
+	//verify count and assert
```

Comment:
add the assertions here

Reply:
Removing integration tests for now since emulator does not support json yet.